index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="A mobile website on a personal portfolio">
+  <meta name="author" content="Promise Chukwu">
+  <meta name="keywords" content="website,portfolio,Promise Chukwu,project">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="index.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <article class="headline">
+  <header>
+    <p class="logo">My Logo</p>
+    <img src="icons/Union.png" id="menu-icon" alt="menu-icon"/>
+  </header>
+  <section class="information">
+    <section class="text">
+      <h1>I'm Promise.<br/>Glad to see<br/>you!</h1>
+      <p id="primary-text">I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p> 
+    </section>
```

Comment:
[OPTIONAL CHANGES]
I see your commitment to writing semantic html tags as you tried to avoid the almighty `div` by all means 😄 
- [ ] Since the parent element is a `section` tag, I'd suggest you use another layout tag like  `article` or any other that fits this portion, just so that `section` is not nesting another `section`.